lib/view_component/slotable.rb

```diff
@@ -98,7 +98,7 @@ def slot(slot_name, **args, &block)
       slot_instance = args.present? ? slot_class.new(**args) : slot_class.new
 
       # Capture block and assign to slot_instance#content
-      slot_instance.content = view_context.capture(&block) if block_given?
+      slot_instance.content = view_context.capture(&block).strip.html_safe if block_given?
```

Comment:
Will `.capture` always return `html_safe?` content?

I'm curious if it'd make sense to check if the captured content is `html_safe?` and only apply `html_safe` if it was already marked as HTML safe.

Reply:
@BlakeWilliams `capture` returns an `ActiveSupport::SafeBuffer`, so I think we're fine to assume that it is safe.